src/lib.rs

```diff
@@ -0,0 +1,93 @@
+extern crate futures;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate quick_error;
+extern crate grpcio as grpc;
+
+pub mod raw;
+pub mod transaction;
+pub mod errors;
+
+use std::path::PathBuf;
+
+use futures::Future;
+
+pub use errors::Error;
+pub use errors::Result;
+
+#[derive(Default, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+pub struct Key(Vec<u8>);
```

Comment:
You can consider `Deref` on this as well.

Reply:
Deref to Vec<u8> added